sport/test/CompetitionAgentTest.scala

```diff
@@ -112,32 +111,12 @@ import scala.concurrent.duration._
     eventually(comps.competitions(0).leagueTable(0).team.id should be("23"))
   }
 
-  it should "recognise placeholder games" in {
```

Comment:
I believe this test can be removed as what was essentially tested here was that `contains` `String` method actually works!